packages/usbutils.rb

```diff
@@ -23,7 +23,11 @@ def self.build
 
   def self.install
     system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
+  end
+
+  def self.postinstall
     FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/share/hwdata/"
     system "wget http://www.linux-usb.org/usb.ids -O #{CREW_DEST_PREFIX}/share/hwdata/usb.ids"
+    puts "It's recommended that you setup a cron job to update this file regularly.".lightblue
```

Comment:
Would it help to provide an example crontab entry?  Also, might be worth mentioning to install cronie (i.e. `crew install cronie`) to get cron working.

Reply:
```
# Example crontab entry to update usb.ids at 3 am daily.
* 3 * * * /usr/local/bin/crew postinstall usbutils
```